File: libraries/hdm/functions/types/request/SnmpGetParameterValuesDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.request.SnmpGetParameterValuesDTO') //< public
//> needs hdm.functions.types.struct.SnmpGetParameterStructDTO
.satisfies('java.io.Serializable')
.protos({
    snmpGetParameterStructs:null, //< private SnmpGetParameterStructDTO[]
    //> public SnmpGetParameterStructDTO[] getSnmpGetParameterStructs()
    getSnmpGetParameterStructs:function(){
        return this.snmpGetParameterStructs;
    },
    //> public void setSnmpGetParameterStructs(SnmpGetParameterStructDTO[] snmpGetParameterStructs)
    setSnmpGetParameterStructs:function(snmpGetParameterStructs){
        this.snmpGetParameterStructs=snmpGetParameterStructs;
    }
})
.endType();

File: libraries/hdm/functions/types/request/SnmpGetParameterValuesDTO.java
package hdm.functions.types.request;

import hdm.functions.types.struct.SnmpGetParameterStructDTO;
import java.io.Serializable;

public class SnmpGetParameterValuesDTO
  implements Serializable
{
  private SnmpGetParameterStructDTO[] snmpGetParameterStructs;

  public SnmpGetParameterStructDTO[] getSnmpGetParameterStructs()
  {
    return this.snmpGetParameterStructs;
  }

  public void setSnmpGetParameterStructs(SnmpGetParameterStructDTO[] snmpGetParameterStructs)
  {
    this.snmpGetParameterStructs = snmpGetParameterStructs;
  }
}